src/mscorlib/src/System.Private.CoreLib.txt

```diff
@@ -854,6 +854,7 @@ BadImageFormat_StreamPositionInvalid = Corrupt .resources file.  The specified p
 BadImageFormat_ResourcesDataInvalidOffset = Corrupt .resources file. Invalid offset '{0}' into data section.
 BadImageFormat_NegativeStringLength = Corrupt .resources file. String length must be non-negative.
 BadImageFormat_ParameterSignatureMismatch = The parameters and the signature of the method don't match.
+BadImageFormat_BadILFormat = Bad IL format.
```

Comment:
Why are we adding this instead of using https://github.com/dotnet/coreclr/pull/10245/files#diff-82c8be167dc0db14d2000fe3135b426dL171?